frontend/src/visualization/filtering/GreyScaleToggle.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { Box, Checkbox, FormControlLabel } from '@material-ui/core';
+import useService from '../../dependency-injection/useService';
+import { EntityColorStore } from '../../stores/colors';
+
+/**
+ * Control that toggles colorization of the graph.
+ */
+export default function GreyScaleToggle(): JSX.Element {
+  const entityColorStore = useService(EntityColorStore);
+
+  // toggle greyScale
+  const [colorize, setGreyScale] = useState<boolean>(
+    !entityColorStore.getGreyScale()
+  );
```

Comment:
use store + consitent naming
```suggestion
  const greyScale = useObservable(entityColorStore.getGreyScale());
```

later (lines 18-21) set it with:

```ts
   entityColorStore.setGreyScale(!greyScale);
```